src/site/content/en/payments/registering-a-web-based-payment-app/index.md

```diff
@@ -0,0 +1,321 @@
+
+{% Label %}[payment handler] payment app's landing page{% endLabel %}
+
+```js
+...
+ await registration.paymentManager.enableDelegations([
+  'shippingAddress', 'payerName'
+]);
+...
+```
+
+You can declare supported delegation options as an array of strings:
+
+* `shippingAddress`: The payment handler can provide a shipping address.
+* `payerName`: The payment handler can provide the payer's name.
+* `payerPhone`: The payment handler can provide the payer's phone number.
+* `payerEmail`: The payment handler can provide the payer's address.
+
+## Debugging a web-based payment app
+
+When developing a web-based payment app frontend, you'll probably jump between
+merchant perspective and payment app perspective. The following debugging tips
+will help your developing experience on Chrome.
+
+### Developing on a local server
+
+Which server do you use for development? Many developers tend to use localhost
+or a company-internal server environment which can be challenging because
+powerful features in the browser tend to require a secure environment (HTTPS)
+and a valid certificate. The Payment Request API and the Payment Handler API are
+no exception and localhosts or company-internal servers usually don't come with
+a valid certificate.
+
+The good news is some browsers, including Chrome, exempt certificates for
+`http://localhost` by default. Also in Chrome, you can exempt the certificate
+requirement by launching a Chrome instance. For example, to exempt the
+requirement from `http://*.corp.company.com`, use the following flags:
+
+* [`--ignore-certificate-errors`](https://chromiumdash.appspot.com/commit/988b56b519836f3d3d94f145ba3554a0c0a7d0a8)
+* [`--unsafely-treat-insecure-origin-as-secure=http://*.corp.company.com`](https://chromiumdash.appspot.com/commit/77a7e1a65d14072149ec4420a0ab523586011d8a)
+
+### Port forwarding a local server
+
+You can port forward the local web server to an Android device using Chrome's
+DevTools and test how it works from a mobile browser. To learn how to do it,
+check out [Access Local
+Servers](https://developers.google.com/web/tools/chrome-devtools/remote-debugging/local-server).
+
+### Remote debugging a website on Android Chrome from desktop DevTools
+
+You can also debug Android Chrome on desktop DevTools. To learn how to do it,
+check out [Get Started with Remote Debugging Android
+Devices](https://developers.google.com/web/tools/chrome-devtools/remote-debugging).
+
+### Payment Handler event logging
+
+DevTools can display Payment Handler API events for easier local development.
+Open DevTools on the merchant context and go to the "Payment Handler" section
+under the "Application" pane.
+
+![Payment Handler event logging](debug-payment-handler.png)
+
+You can capture events sent to the service worker that handles payments.
+
+{% Aside 'caution' %}
+Make sure that "Show events from other domains" is enabled.
+{% endAside %}
```

Comment:
```suggestion
```
This is important information for the debugging workflow and therefore shouldn't be in an aside. Please add it into the paragraph above where you provide the debugging workflow instructions